app/src/main/java/com/example/theperiodpurse/ui/calendar/CalendarScreen.kt

```diff
@@ -266,12 +290,13 @@ val previewTrackedSymptoms = Symptom.values().asList()
 
 @RequiresApi(Build.VERSION_CODES.O)
 @Composable
-fun CalendarScreenLayout(navController: NavController) {
+fun CalendarScreenLayout(calendarUIState: CalendarUIState, navController: NavController) {
     // Contains the swappable content
     ThePeriodPurseTheme {
         val bg = painterResource(R.drawable.colourwatercolour)
 
         var selectedDate by remember { mutableStateOf<LocalDate?>(null) }
+        var activeSymptom: Symptom? by remember { mutableStateOf(null) }
```

Comment:
This state is actually not remembered when navigating between pages. You have two options: 

1. store the state in a higher order component that's not affected by navigation, or
2. store the state in the `calendarUIState` then update it with `calendarViewModel`

This should be true for `selectedDate` as well.

Reply:
Right, that'll fix the issue of not remembering the active symptom. Fixed! Also removed `selectedDate` because it was not doing anything.